README.md

```diff
@@ -83,15 +83,22 @@ Create a new folder under `packages` if it is a common library or create folder
 
 Make sure to name the package as `@cumulus/package-name`.
 
-## Publishing to NPM
+## Versioning
+
+We use a global versioning approach, meaning version numbers in cumulus are consistent across all packages and tasks. We use Lerna to manage our versioning. Any change, whether patch, minor, or major, will force lerna to increment the version of all packages.
+
+### Publishing to NPM
 
     $ lerna publish
 
+To specify the level of change for the new version
```

Comment:
Do you always want to specify the level of change? What is the default? 

For the levels of change, how do you know which one to choose? Are we using premajor/preminor/prepatch/prerelease?

Reply:
<img width="456" alt="screen shot 2018-01-19 at 9 36 10 am" src="https://user-images.githubusercontent.com/12213601/35155665-61341e8c-fcfc-11e7-80b1-160aad7f12b2.png">
I just tested, without putting a flag on, it will give you the option instead of defaulting to anything specific. I have seen cases where I was working with the --canary tag without the --cd-version tag where it defaulted to minor. 

I guess the level of change will have to be up to the author. Right now we're using prerelease ("beta") but when the message-updates branch is merged we'll go right to 1.0.0. I could remove the "premajor/preminor/prepatch/" options to reduce confusion.

Reply:
Ok thanks! I think removing the unnecessary ones makes it more clear.